src/main/resources/assets/tconstruct/lang/en_us.json

```diff
@@ -341,6 +341,9 @@
     "item.tconstruct.efln_ball": "EFLN",
     "item.tconstruct.efln_ball.tooltip": "Explodes! Safe for mining",
 
+    "item.tconstruct.quartz_shuriken": "Quartz Shuriken",
+    "item.tconstruct.flint_shuriken": "Flint Shuriken",
```

Comment:
Maybe want tooltips? Could do something like "Throwable weapon with high knockback" for flint and "Throwable weapon with higher damage" for quartz. To make the differences more visible.